packages/integration-tests/src/test-projects/conversation-handler/amplify/backend.ts

```diff
@@ -0,0 +1,17 @@
+import { defineBackend } from '@aws-amplify/backend';
+import { ConversationHandlerFunction } from '@aws-amplify/ai-constructs/conversation';
+import { auth } from './auth/resource.js';
+import { data } from './data/resource.js';
+
+const backend = defineBackend({ auth, data });
+
+const stack = backend.createStack('conversationHandlerStack');
+
+new ConversationHandlerFunction(stack, 'conversationHandlerFunction', {
```

Comment:
In real life it will be - to appsync resolver.

in the test, we're mocking the upstream appsync component so that we can (kind of) e2e test this in isolation while other components are still built.
I.e.
The test sends event instead of appsync resolver [here](https://github.com/aws-amplify/amplify-backend/blob/cf7778fa3877f3650743b793283b3f4cae362407/packages/integration-tests/src/test-project-setup/conversation_handler_project.ts#L145-L176) and we assert that response was published [here](https://github.com/aws-amplify/amplify-backend/blob/cf7778fa3877f3650743b793283b3f4cae362407/packages/integration-tests/src/test-project-setup/conversation_handler_project.ts#L220-L224)


This might get replaced with simpler approach later when other components (related to schema builder changes) mature. But for now this is what can be done.